infrastructure/ai/prompt_template_loader.py

```
@@ -0,0 +1,307 @@
+                    )
+
+                # 校验 @output
+                header_output = header.get("output", "text")
+                if header_output != meta.output_format:
+                    warnings_list.append(
+                        f"{template_name}: @output='{header_output}' "
+                        f"与 manifest output_format='{meta.output_format}' 不一致"
+                    )
+
+            except Exception:
+                pass  # 模板不存在不在此处报错
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)